src/index.ts

```diff
@@ -42,7 +43,10 @@ const getUrlString = (url: string): string | null => {
   }
 }
 
-function remarkEmbedder({transformers, cache}: RemarkEmbedderOptions) {
+const remarkEmbedder: Attacher<RemarkEmbedderOptions[]> = ({
```

Comment:
Is it necessary to have  an array of type `RemarkEmbedderOptions` or can we just have `RemarkEmbedderOptions` type?

Reply:
I'm checking this, looks like something is still missing to be a correctly typed remark plugin, will update when done!